src/ducks/components/intents/OpenAppsIntent.jsx

```diff
@@ -0,0 +1,181 @@
+import {
+  APP_TYPE,
+  getAppBySlug,
+  getRegistryApps,
+  installAppFromRegistry,
+  initApp
+} from 'ducks/apps'
+import OpenAppsIntentRoutes from 'ducks/components/intents/OpenAppsIntentRoutes'
+import compose from 'lodash/flowRight'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { HashRouter } from 'react-router-dom'
+
+import { withClient } from 'cozy-client'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import IntentHeader from 'cozy-ui/transpiled/react/IntentHeader'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+
+export class OpenAppsIntent extends Component {
+  state = {
+    hasWebappSucceed: false,
+    hasKonnectorSucceed: false
+  }
+
+  componentDidMount() {
+    this.props.initAppIntent()
+  }
```

Comment:
I propose to remove this component since i suggest to use the previous intent. But if you don't to in that direction, we should not create a class component anymore.

Reply:
Vu ensemble, je vais retravailler le composant `InstallAppsIntent`(`INSTALL`), pour qu'il s'adapte aux nouveaux besoins.